openstack/keymanager/v1/containers/results.go

```diff
@@ -230,3 +230,27 @@ func ExtractConsumers(r pagination.Page) ([]Consumer, error) {
 	err := (r.(ConsumerPage)).ExtractInto(&s)
 	return s.Consumers, err
 }
+
+type secretRefResult struct {
```

Comment:
Where is this being used?

Reply:
You're right, this was a copy-paste. Will remove it.